src/Gateway/PayplugResponse.php

```diff
@@ -110,11 +110,11 @@ public function process_payment($resource, $is_payment_with_token = false)
 				}
 			}
 
-		} else if (($this->gateway->id == "payplug") || ($this->gateway->id == "bancontact")) {
```

Comment:
instead of repeating all of this, you could 
ucfirst($this->gateway->id) on the logs
also probably it's readable if you separate it by methods

Reply:
Done